roles/pulp_common/vars/main.yml

```diff
@@ -31,7 +31,7 @@ __pulp_common_pulp_settings_defaults:
       PASSWORD: pulp
   private_key_path: "{{ pulp_certs_dir }}/token_private_key.pem"
   public_key_path: "{{ pulp_certs_dir }}/token_public_key.pem"
-  token_server: "https://{{ ansible_facts.fqdn }}/token"
+  token_server: "{{ pulp_webserver_disable_https | default(false) | ternary('http', 'https') }}://{{ ansible_facts.fqdn }}/token"
```

Comment:
@ipanova this is what bit you the other day

Reply:
Nice!

Reply:
yes! thank you :woman_dancing: